### ISSUE TITLE
Test Set

### ISSUE THREAD

**Issue by WorldHellow** [abhika-m/FAVA#1]
Brilliant work. Can the authors please share the test set. On huggingface only training data is available.

**Comment by abhika-m** [comment]
Hi! Our annotations.json file is what we used to test hallucination detection [which can be found here on huggingface](https://huggingface.co/datasets/fava-uw/fava-data/raw/main/annotations.json) and to test edits we run fava on FActScore prompts/outputs and loading FActScore's data set [can be found here](https://github.com/shmsw25/FActScore).
